mergefs.go

```diff
@@ -1,8 +1,11 @@
 package mergefs
 
 import (
+	"errors"
+	"github.com/rs/zerolog/log"
```

Comment:
I think it's better to remove zerolog. Many people use different logging libraries other than zerolog. Better not to force them to have another one. Also, it doesn't seem like there's a good enough reason to need it

Reply:
I can change to the standard `Go` `log` library - I think there's a good reason to have it since it's nice to have also Debug options. Playing around with filesystems could be really tricky, so better to have them there - no harm :)